src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/bookstore?useSSL=false";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "1234");
```

Comment:
Be cautious with hardcoded database credentials. Consider using environment variables or a secure configuration management system to store sensitive information, especially in a production environment.